content/blog/partnership-with-junior-guru.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "Partnership With junior.guru"
+description = "This next episode of our interview series includes a chat with Daria Grudzien - one of CyberMagnolia's Co-Founders."
+author = "Daria Grudzien"
+date = "2021-03-18"
+tags = ["partnership"]
+[[images]]
+  src = "/img/jg_partnership.jpeg"
+  alt = "Photo"
+  stretch = "stretchH"
++++
+
+Today we proudly announce of first partnership with an inspiring tech community **junior.guru** created by Honza Javorek.
```

Comment:
"we proudly announce of first partnership" sounds a bit weird to me, I believe the formulation is correct when announce is a noun but in this case as we use it as a verb so I would use maybe announce our first partnership... not sure, just feeling.

Reply:
I see, let me fix this up.